lib/events.js

```diff
@@ -21,8 +21,22 @@
 
 'use strict';
 
-const { Math, Object, Reflect } = primordials;
-const apply = Reflect.apply;
+const {
+  Math: {
+    min: MathMin
+  },
+  Object: {
+    defineProperty: ObjectDefineProperty,
+    getPrototypeOf: ObjectGetPrototypeOf,
+    create: ObjectCreate,
+    keys: ObjectKeys,
+  },
+  Reflect: {
+    apply: ReflectApply,
+    ownKeys: ReflectOwnKeys,
+  }
+} = primordials;
+const apply = ReflectApply;
```

Comment:
The variables `ReflectApply` and `apply` are the same thing. I think one of them can be removed.

Reply:
That's right. I have changed it. Thanks!